tests/integration/test_client.py

```diff
@@ -618,7 +620,7 @@ async def test_transact_write_items_put(client: Client, table: TableName) -> Non
         Put(table=table, item={"h": "h", "r": str(i), "s": "initial"}) for i in range(2)
     ]
     response = await client.transact_write_items(items=puts)
-    assert not response
+    assert response is None
```

Comment:
IMO, `not X` is more Pythonic.
In this case, however, since `transact_xxx` never returns anything, the assertion is best removed.